src/videouplinkbandwidthpolicy/SimulcastUplinkPolicy.ts

```diff
@@ -195,7 +212,9 @@ export default class SimulcastUplinkPolicy implements VideoUplinkBandwidthPolicy
       false
     );
     this.videoIndex = videoIndex;
-    this.newQualityMap = this.calculateEncodingParameters(numParticipantsChanged);
+    this.newQualityMap = this.calculateEncodingParameters(
+      numSendersChanged || numParticipantsChanged
```

Comment:
nit: a bit confusing with the naming 

```
private calculateEncodingParameters(
    numParticipantsChanged: boolean
    numSendersChanged: boolean
  ): Map<string, RTCRtpEncodingParameters> {
``` 

I could think of something super verbose like `p2pAndMultipointCallTransitioned` and not great either.